### PR TITLE
added new non-zero aspects for books and enchanted books

### PR DESCRIPTION
**Pull request by KiloJoel**

### CODE REVIEW

File: scripts/Thaumcraft-07-Aspect-Adds.zs [hunk 1]
```diff
@@ -6,6 +6,16 @@
...
 
+// --- Minecraft ---
+
+
+// --- Book
+mods.thaumcraft.Aspects.set(<minecraft:book>, "pannus 1, cognitio 1");
+
+// --- Enchanted book
+mods.thaumcraft.Aspects.set(<minecraft:enchanted_book>, "pannus 1, cognitio 1, metallum 1");
+
```
Comment (leagris) [inline scripts/Thaumcraft-07-Aspect-Adds.zs:17]: Is missing 2×`volatus` and 2×`praecantatio`

Should be:
```zs
mods.thaumcraft.Aspects.set(<minecraft:enchanted_book>, "pannus 1, cognitio 1, volatus 2, praecantatio 2, metallum 1");
```
```

**Review by leagris — CHANGES_REQUESTED** [review]
Please fix enchanted book aspects.

File: scripts/Thaumcraft-07-Aspect-Adds.zs (hunk 1 above)
Comment (KiloJoel) [reply scripts/Thaumcraft-07-Aspect-Adds.zs:17]: It also gets aspects from the enchantments it has on it, if you look at a creatively spawned one with no enchantment, it has only the three I added.

Comment (leagris) [reply scripts/Thaumcraft-07-Aspect-Adds.zs:17]: ok then

**Review by leagris — APPROVED** [review]

**Review by Dream-Master — APPROVED** [review]

**Pull request merged**